configure.ac

```diff
@@ -1,3 +1,4 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
I see we have some files that use GPL-3.0 License.

From the fast check I did, I believe GPL-3.0 license requires including the full text of the license in repository.
I believe we don't have that.

Reply:
SPDX is fine for GPL 3.  See https://spdx.org/licenses/

Reply:
I meant dedicated `COPYING` or `LICENSE` file like we have for LGPL and GPL 2.0.

GPL 3.0 states that a copy of license should be included along with the program.


I'm not an expert on licensing, but from what I found [online](https://github.com/david-a-wheeler/spdx-tutorial/tree/master#:~:text=If%20you%27re%20developing%20open%20source%20software%20(OSS)%20you%20should%20still%20include%20the%20full%20legal%20text%20of%20your%20license(s).%20Include%20them%20at%20the%20top%20level%20of%20the%20project%20repository%2C%20with%20the%20conventional%20all%2Dupper%2Dcase%20filename%20LICENSE%20or%20COPYING%20(optionally%20with%20a%20.txt%20or%20.md%20extension).) a copy still should be attached.

Reply:
@peluse please advice :)
 SPDX is saying:
> The SPDX License List includes a standardized short identifier, the full name, the license text, and a canonical permanent URL for each license and exception.

but the point Mateusz raised is also valid.

Reply:
From our resident expert:

You should add the text of the GPLv3 license in the repo somewhere. At toplevel you already have COPYING and COPYING.LIB.  Probably you can rename these to
LICENSE.GPLv2 and LICENSE.LGPL and add a new LICENSE.GPLv3. Or create a toplevel
LICENSES directory and have them all in there.

Reply:
OK, I added LICENSES dir and verification of existence of license file. I left COPYING and COPING.lib as it is because I want to keep it obvious for developers. Mixed licenses are internal thing.